private-distributors-list.md

```diff
@@ -6,6 +6,16 @@ security issues.
 
 ### Embargo Policy
 
+The patch files and any derived patched binaries are included in the embargo, 
+and should not be distributed or made available to external parties before the 
+public disclosure is made.
+
+However, a fully-hosted patched kube-apiserver can be deployed 
+prior to the embargo lift date if users do not have direct access to the binary.
+
+However, a fully-hosted patched NON_API_SERVER_COMPONENT can be deployed 
+prior to the embargo lift date if all users of the environment are internal to the
```

Comment:
```suggestion
prior to the embargo lift date only if all users of the environment are internal to the
```
Not sure on the suggestion I'm proposing but do we want to make this a little more strongly worded? Is the scope of  "environment" just the cluster?

Reply:
Hmm, yes?  Not sure on wording either.

Reply:
updated to `only if all users with access to the components are internal to the Kubernetes distributor` lmk wyt